### DISCUSSION TITLE
Question on simple fraction evaluation

### DISCUSSION THREAD

**Discussion by DougM52** [wxMaxima-developers/wxmaxima#1492] [Q&A]
Using wxMaxima 18.02.0

Not sure why (24 - 3) * (4 + 28) / 14 evaluates to (3* (32))/2 but 21 * 32 / 14 evaluates to 48.  The two is tests I would think would be true.  How do I get the  (3* (32))/2 to evaluate to 48 as does 21 * 32 / 14 ?

(%i104)	/*
	(b)
	(24 - 3) * (4 + 28) / 14;
	(24 - 3)/14 * (4 + 28)
	(3 / 2) * 32
	3*16;
	48;
	*/
	(24 - 3) * (4 + 28) / 14;
	21 * 32 / 14;
	is ((24 - 3) * (4 + 28) / 14 = 48);
	is (21 * 32 / 14 = 48);
(%o101)	(3* (32))/2
(%o102)	48
(%o103)	false
(%o104)	true

**Comment by DougM52** [comment]
It appears to work if I trim spaces out of the expressions.

(%i9)	/*
	(b)
	(24-3)*(4+28)/14;
	(24-3)/14*(4+28)
	(3/2)*32
	3*16;
	48;
	*/
	is ((24-3)*(4+28)/14=48);
	is (21*32/14=48);
(%o8)	true
(%o9)	true

**Reply by gunterkoenigsmann** [reply]
Then you use an old wxMaxima Version and have accidentally used nonbreakable spaces.
